ambry-frontend/src/main/java/com.github.ambry.frontend/AmbrySecurityService.java

```diff
@@ -101,7 +104,7 @@ public void postProcessRequest(RestRequest restRequest, Callback<Void> callback)
       throw new IllegalArgumentException("RestRequest or Callback is null");
     }
     // check preconditions for DELETE request
-    if (restRequest.getRestMethod() == RestMethod.DELETE) {
+    if (restRequest.getRestMethod() == RestMethod.DELETE || restRequest.getRestMethod().equals(RestMethod.PUT)) {
```

Comment:
minor: you can use == here since it's an enum (or change the `== RestMethod.DELETE` to `.equals(RestMethod.DELETE)` for consistency)

Reply:
Done